frontend/src/features/admin-form/settings/components/FormIssueNotificationToggle.tsx

```diff
@@ -0,0 +1,38 @@
+import { useCallback, useMemo } from 'react'
+import { Skeleton } from '@chakra-ui/react'
+
+import Toggle from '~components/Toggle'
+
+import { useMutateFormSettings } from '../mutations'
+import { useAdminFormSettings } from '../queries'
+
+export const FormIssueNotificationToggle = (): JSX.Element => {
+  const { data: settings, isLoading: isLoadingSettings } =
+    useAdminFormSettings()
+
+  const hasIssueNotification = useMemo(
+    () => settings && settings?.hasIssueNotification,
+    [settings],
+  )
+
+  const { mutateFormIssueNotification } = useMutateFormSettings()
+
+  const handleToggleIssueNotification = useCallback(() => {
+    if (!settings || isLoadingSettings || mutateFormIssueNotification.isLoading)
```

Comment:
Same comment for this part with the repeated checks! When you make a decision of which one to use, remember to update the dependencies below too.

Reply:
Ah I can't do it there @LinHuiqing, cause it will cause typing issues on the next line `const nextHasIssueNotification = !settings.hasIssueNotification`

as settings is `FormSettings | undefined`

Reply:
Ah got it, because there's no "settings is not undefined" check here.